### PR TITLE
More on python 3 compatibility

### PR DESCRIPTION
**Pull request by magmax**
Making errors visible and avoiding "catch pokémon" anti-pattern.
Making it to work in python 2 and 3 (now for sure).

I tried to catch `ConnectionRefusedError`, but it is just in python3. It seems both python 2 and 3 has socket.error and `ConnectionRefusedError` depends on it.

### CODE REVIEW

**Comment by magmax** [comment]
Please, wait. Another issue with line.partition. I will solve it as soon as I could.

**Comment by magmax** [comment]
It should be working now.

**Comment by magmax** [comment]
Now it is fine.

**Comment by paetling** [comment]
Taking A look

File: zookeeper_dashboard/common.py [hunk 1]
```diff
@@ -3,4 +3,13 @@
...
+    return (
+        os.getenv('ZOOKEEPER_SERVERS')
+        or getattr(settings,'ZOOKEEPER_SERVERS')
+    )
+
+def get_zookeeper_servers_as_list():
+    return get_zookeeper_servers().split(',')
+
+def get_zookeeper_server(id):
+    return get_zookeeper_servers_as_list()[int(id)]
```
Comment (paetling) [inline zookeeper_dashboard/common.py:15]: Why have the function convert to an int for you vs making id -> index and specifying that an int must be passed in?  Convenience?

Comment (magmax) [inline zookeeper_dashboard/common.py:15]: Yes, just convenience.

**Comment by paetling** [comment]
👻 🐮

**Pull request merged**